packages/web/docs/theme.config.tsx

```diff
@@ -24,5 +24,5 @@ export default defineConfig({
       </>
     );
   },
-  siteName: 'HIVE',
+  siteName: 'Hive',
```

Comment:
```suggestion
  siteName: 'HIVE',
```

should be `HIVE` because og-images accept UPPER_CASE products